typings/index.d.ts

```diff
@@ -586,8 +586,10 @@ export interface A11yRemovalMessage<Item> {
 }
 
 export interface UseMultipleSelectionGetSelectedItemPropsOptions<Item>
-  extends GetItemPropsOptions<Item>,
-    GetPropsWithRefKey {}
+  extends React.HTMLProps<HTMLElement>, GetPropsWithRefKey {
```

Comment:
can we have a separate one for this, similar to the one below but only with `index` and `selectedItem` ?
```ts
export interface GetItemPropsOptions<Item>
  extends React.HTMLProps<HTMLElement> {
  index?: number
  item: Item
  isSelected?: boolean
  disabled?: boolean
}
```

Reply:
I'm not sure if I understand you. That's what I did. `UseMultipleSelectionGetSelectedItemPropsOptions` is no longer extending the wrong `GetItemPropsOptions` and instead has `index` and `selectedItem` props of its own

Reply:
Actually it doesn't matter since it will only be used only here.

I was suggesting to have a separate GetSelectedItemPropsOptions with index and selectedItem and make UseMultipleSelectionGetSelectedItemPropsOptions extend that, instead of having index and selectedItem directly in UseMultipleSelectionGetSelectedItemPropsOptions .